index.html

```diff
@@ -17,46 +23,60 @@
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
+                document.forms.inputs.result.value = parseInt(numberOne) + parseInt(numberTwo);
             },
             subtractNumbers: function () {
-                
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) - parseInt(numberTwo);
+            },
+            multiplyNumbers: function () {
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) * parseInt(numberTwo);
+            },
+            divideNumbers: function () {
+                var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) / parseInt(numberTwo);
             }
         }
 
     </script>
 
 </head>
 
```

Comment:
This is good. Note that what is in the `style` here on the `body` tag can be in the `style` tag instead where the `body` selector is defined and currently has the margin set. For example:

```
body {
  margin: 10px;
  background-color: rgb(148, 109,221);
}

```